var/spack/repos/builtin/packages/portage/package.py

```diff
@@ -36,7 +36,9 @@ class Portage(CMakePackage):
     variant("openmp", default=False, description="Enable on-node parallelism using OpenMP")
     variant("cuda", default=False, description="Enable GPU parallelism using CUDA")
 
-    depends_on("cxx", type="build")  # generated
+    depends_on("c", type="build")
+    depends_on("cxx", type="build")
+    depends_on("fortran", type="build")
```

Comment:
Why fortran?

Reply:
The issue is that the wonton package installed CMake seems to always require Fortran language. I've told them that they should not do that, but in the meantime we need it. An alternative would be to have `type=("build", "link")`, but when I did that my entire concretization changed.

Reply:
@hobywan Let's work with @rbberger on this.

Reply:
Looking at the CMake file, Fortran is enabled only when lapacke is on. That should be reflected in the Spack recipe.